lib/lego/lego_uart.h

```diff
@@ -474,6 +474,9 @@ typedef enum {
     /** Technic Medium Angular Motor */
     LUMP_TYPE_ID_TECHNIC_M_ANGULAR_MOTOR  = 75,
 
+    /** Technic Large Angular Motor */
+    LUMP_TYPE_ID_TECHNIC_M_ANGULAR_MOTOR  = 76,    
```

Comment:
`M` probably needs to be `L` here. The identical name is causing conflicts in the build.

Thanks for the PR!

Reply:
arg... forget to edit